src/modules/ContentCard.js

```diff
@@ -52,6 +52,18 @@ export default class ContentCard extends Component {
     return children;
   }
 
+  _renderWatchIcon (size) {
```

Comment:
I've created a PR in grommet-core with the Watch icon and tested it with this toolkit PR:
https://github.com/grommet/grommet/pull/783